sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/TokenResult.java

```diff
@@ -81,7 +82,10 @@ public Date expiresOn() {
         try {
             return JSONDateUtils.toDate(this.expiresOn);
         } catch (JSONException ex) {
-            AppCenterLog.error(Constants.LOG_TAG, "Unable to convert '" + this.expiresOn + "' to ISO 8601 Date format ");
+            AppCenterLog.error(
+                    LOG_TAG, String.format(
+                            "Unable to convert '%s' to ISO 8601 Date format ",
+                            this.expiresOn));
```

Comment:
Sometimes `this` is used in this file, sometimes not.
Since usually we use `mField` in all objects except models, we don't need to use `this`, maybe just remove it unless strictly required.